ds-prep-python-variables/main.py

```diff
@@ -0,0 +1,36 @@
+from types import ClassMethodDescriptorType
+
+
+first_name = 'ravleen'
+last_name = "kaur"
+full_name = first_name + " " + last_name
+print(full_name)
+height_in_inches = 66
+x = type(height_in_inches)
+print(height_in_inches)
+print(x)
+y = height_in_inches_float = 66.0
```

Comment:
The value assigned to `height_in_inches_float` should be derived from the `height_in_inches` variable, not hard-coded:

---

![image](https://user-images.githubusercontent.com/16143771/148597109-eef81da4-434e-446d-9087-6160b3b312e7.png)